resolwe_bio/processes/variant_calling/variants_to_table.py

```diff
@@ -0,0 +1,123 @@
+"""Run GATK VariantsToTable."""
+
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class GatkVariantsToTable(Process):
+    """Run GATK VariantsToTable.
+
+    This tool extracts specified fields for each variant in a VCF file
+    to a tab-delimited table, which may be easier to work with than a VCF.
+    For additional information, please see
+    https://gatk.broadinstitute.org/hc/en-us/articles/360036711531-VariantsToTable
+    """
+
+    slug = "variants-to-table"
+    name = "GATK VariantsToTable"
+    category = "GATK"
+    process_type = "data:variants:vcf:variantstotable"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/dnaseq:6.2.0"}
+        },
+        "resources": {
+            "cores": 2,
+            "memory": 16384,
+            "storage": 200,
+        },
+    }
+    data_name = "Variants in table"
+
+    class Input:
+        """Input fields for GATK VariantsToTable."""
+
+        vcf = DataField("variants:vcf", label="Input VCF file")
+
+        vcf_fields = ListField(
+            StringField(),
+            label="Fields from VCF file to include",
+            description="The name of a standard VCF fields or "
+            "an INFO fields to include in the output table.",
+            default=[
+                "CHROM",
+                "POS",
+                "ID",
+                "REF",
+                "ALT",
+            ],
+        )
+        advanced = BooleanField(
+            label="Show advanced options",
+            description="Inspect and modify parameters.",
+            default=False,
+        )
+
+        class AdvancedOptions:
+            """Advanced options."""
+
+            gf_fields = ListField(
+                StringField(),
+                label="Include FORMAT/sample-level fields",
+                default=["GT"],
+            )
+            split_alleles = BooleanField(
+                label="Split multi-allelic records",
```

Comment:
IMO it would be nice to have this full description

Split multi-allelic records into multiple lines
By default, a variant record with multiple ALT alleles will be summarized in one line, with per alt-allele fields (e.g. allele depth) separated by commas. This may cause difficulty when the table is loaded by an R script, for example. Use this flag to write multi-allelic records on separate lines of output. Fields that are not allele-specific will be duplicated.